tutorials/commands/flux-proxy-command.rst

```diff
@@ -0,0 +1,194 @@
+
+This tutorial will introduce the :core:man1:`flux-proxy` command, which can be used to interact with subinstances.
+
+-----------------------
+Starting a Subinstance
+-----------------------
+
+First, let's create a subinstance to work with.  The most common way is through ``flux mini batch``.  We'll launch a subinstance and run a few jobs under that instance, we then call ``flux job status`` to wait on those jobs to finish.
+
+.. code-block:: sh
+
+   #!/bin/sh
+   #filename: subinstance-jobs.sh
+
+   jobid1=`flux mini submit -n1 sleep inf`
+   jobid2=`flux mini submit -n1 sleep inf`
+   jobid3=`flux mini submit -n1 sleep inf`
+   jobid4=`flux mini submit -n1 sleep inf`
+   flux job status ${jobid1} ${jobid2} ${jobid3} ${jobid4}
+
+.. code-block:: console
+
+   > flux mini batch -n4 ./subinstance-jobs.sh
+   f2NRGXaP
+
+When we run ``flux jobs`` you'll notice that it returns a listing for the subinstance, not the jobs run within the subinstance.
+
+.. code-block:: console
+
+   > flux jobs
+          JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+       f2NRGXaP achu     subinstan+  R      4      1   1.007m opal186
+
+.. note::
+
+   Depending on your terminal settings, the subinstance may be colored blue, indicating it is a subinstance.
+
+We can list the jobs within the subinstance by listing jobs recursively with `--recursive``.
+
+.. code-block:: console
+
+    > flux jobs --recursive
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+        f2NRGXaP achu     subinstan+  R      4      1   1.544m opal186
+
+    f2NRGXaP:
+         fvhpz5V achu     sleep       R      1      1   1.493m opal186
+         foyvExo achu     sleep       R      1      1   1.497m opal186
+         fh2fcK1 achu     sleep       R      1      1   1.502m opal186
+         fa6txwZ achu     sleep       R      1      1   1.506m opal186
+
+But how could we submit a job to that subinstance?
+	 
+-----------------------
+Connect to the Instance
+-----------------------
+
+The easiest way to operate with a subinstance is to use ``flux-proxy``.  You can pass it the subinstance jobid and the command you want to run against that subinstance.  Lets submit a job via ``flux mini submit`` and list it via ``flux jobs``.
+
+.. code-block:: console
+
+    > flux proxy f2NRGXaP flux mini submit hostname
+    f2sW9cPyZ    
+
+    > flux proxy f2NRGXaP flux jobs
+            JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+        f2sW9cPyZ achu     hostname    S      1      -        - 
+          fvhpz5V achu     sleep       R      1      1   4.143m opal186
+          foyvExo achu     sleep       R      1      1   4.148m opal186
+          fh2fcK1 achu     sleep       R      1      1   4.152m opal186
+          fa6txwZ achu     sleep       R      1      1   4.157m opal186
+
+Notice that we're running ``flux jobs`` against the subinstance itself.  We don't have to specify the ``--recursive`` option to list the jobids anymore.
```

Comment:
More interesting might be to submit more work to the instance (this is where `flux job wait --all` would come in handy)